### DISCUSSION TITLE
Introduction (Non-) Anonymous Voting

### DISCUSSION THREAD

**Discussion by BenedictHomuth** [inovex/scrumlr.io#4100] [Proposals]
# Community Feature Request
To add the possibility of seeing who voted for which card.

Since for now, we only allow for anonymous votings, it could be beneficial to introduce Anonymous / Non-Anonymous voting-modes.

# How it could look like:

## Start a voting with toggle for mode selection
![voting_no_name](https://github.com/inovex/scrumlr.io/assets/56362368/9cbe0666-6a02-4e86-b2b1-a089e379013c)

## After a voting is concluded, you can hover over the votes to see who voted. 
(Could also include how many votes per user)
![voting](https://github.com/inovex/scrumlr.io/assets/56362368/709a3d6a-a931-4b57-9826-6b24a3ccf03c)

**Comment by brandstetterm** [comment]
Great idea 👍🏼 Since this would require some changes in our backend and client-side redux store & UI, I would suggest that we come back to your idea after we have realized our currently planned milestones. However, I would definitely make a note of the idea and keep the discussion open so that other people can give their opinion.

**Comment by Resaki1** [comment]
I like the idea as well. Some thoughts:

- anonymous voting should be enabled by default
- participants should be able to see wether the voting is anonymous or not
- when implementing the feature, we need to keep in mind that users can give more than one vote per note

**Comment by SelinaBuff** [comment]
Like the idea too! I will open a new issue for our Backlog.
